extension/webviews/types.ts

```diff
@@ -1,3 +1,5 @@
+import { CopilotRole } from './enums';
```

Comment:
even tho we are using this import in CopilotMessage for some reason ts throws an unused import error. 
```
(!) Plugin typescript: @rollup/plugin-typescript TS1371: This import is never used as a value and must use 'import type' because 'importsNotUsedAsValues' is set to 'error'.
webviews/state/store.ts: (2:1)
 import { type Project, type Organization, type Thread, CopilotMessage } from '../types';   ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
webviews/types.ts: (1:1)
 import { CopilotRole } from './enums';
```